zenodo/modules/frontpage/templates/zenodo_frontpage/index.html

```diff
@@ -44,7 +44,7 @@
         {%- set publication_date = r.publication_date|from_isodate -%}
         {%- set creation_date = r._created|from_isodatetime -%}
         {%- set embargo_date = r.embargo_date|from_isodate -%}
-        {%- set object_type = r.resource_type|objecttype %}
+        {%- set object_type = r.resource_type.to_dict()|objecttype %}
```

Comment:
I moved the check to ObjectType. Seems they removed the get method on purpose earlier. https://github.com/elastic/elasticsearch-dsl-py/commit/62ce0fc3684709e1370e00554b706eda53220075